js/docs.js

```diff
@@ -222,7 +230,8 @@ var docs = {
     'ua': '<p>Визначає названі grid зони.</p><p><code>&lt;grid-name&gt;</code></p>',
     'no': '<p>Spesifiserer navngitte områder på rutenettet.</p><p><code>&lt;grid-name&gt;</code></p>',
     'pt-pt': '<p>Especifica as áreas nomeadas da grelha.</p><p><code>&lt;grid-name&gt;</code></p>',
-    'bg': '<p>Определя наименованите зони от решетката.</p><p><code>&lt;grid-name&gt;</code></p>'
+    'bg': '<p>Определя наименованите зони от решетката.</p><p><code>&lt;grid-name&gt;</code></p>',
+    'he': '<p>קובע את אזורי הסריג שנקבעו.</p><p><code>&lt;grid-template-rows&gt; / &lt;grid-template-columns&gt;</code></p>',
```

Comment:
Can you remove this change from `<grid-name>` to `<grid-template-rows> / <grid-template-columns>`?

Reply:
done as well I believe